### PR TITLE
review of the new execution text

### PR DESCRIPTION
**Pull request by boucadair**
The text is too specific to an action type. I'm afraid some "more" words are needed to explain why the focus on this aspect.

### CODE REVIEW

**Comment by zhangli-abcd** [comment]
Actually this part is contributed by Tony Li, and describes about the actions to the topoloy changes. I think it should be put in this section, but yes, this section should also be enriched with other considerations. How about we add two sub-section here? one is schedules without topology changes, and another is schedules with topology changes.

File: draft-wqk-control-schedule-framework.md [hunk 1]
```diff
@@ -428,13 +428,13 @@ TODO
 
 ### Executing Schedule
 
-Schedules execution means that a device allocates and deallocates resources at a specified
-time point. However, when a schedule's execution affects the network topology, the addition
-and deletion of the topology need to be considered separately.
+Schedules execution means that a component (e.g., device) undertakes an action (e.g., allocates and deallocates resources) at specified
+time points. When a schedule's execution affects the network topology, the addition
```
Comment (QiufangMa) [inline draft-wqk-control-schedule-framework.md:432]:
```suggestion
time points. The schedule executor should fully understand the consequences of the schedule execution. For example, when a schedule's execution affects the network topology, the addition
```

**Review by QiufangMa — COMMENTED** [review]
agree that it is too specific.

File: draft-wqk-control-schedule-framework.md [hunk 2]
```diff
@@ -428,13 +428,13 @@ TODO
...
-time point. However, when a schedule's execution affects the network topology, the addition
-and deletion of the topology need to be considered separately.
+Schedules execution means that a component (e.g., device) undertakes an action (e.g., allocates and deallocates resources) at specified
+time points. The schedule executor should fully understand the consequences of the schedule execution. For example, when a schedule's execution affects the network topology, the addition
+and deletion of the topology need to be considered carefully.
 
-There is very little reaction that the network should do in response to a topological addition.
-A link coming up or a node joining the topology should not have any functional change until the
-change is proven to be fully operational. The routing table or paths could be pre-computated
+There is very little reaction that a network should do in response to a topological addition.
```
Comment (boucadair) [inline draft-wqk-control-schedule-framework.md:435]:
```suggestion
```

**Comment by boucadair** [comment]
Thanks for clarifying @zhangli-abcd.

I do see actions with topology changes as an example inter-dependency implications when executing scheduled actions. I think this text should be presented as an example. I tried to soften some questionable claims (but not all of them).

We can merge this PR and reconsider the text in the next iteration.

**Review by zhangli-abcd — COMMENTED** [review]
Great

**Pull request merged**